app/controllers/comments_controller.rb

```diff
@@ -1,4 +1,6 @@
 class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
```

Comment:
- [ ] In order to use  **CanCanCan** for Comment authorization in `Comment Controller`, add this line `load_and_authorize_resource` on the top of your controller 👇🏻 

   ```ruby
     class CommentsController < ApplicationController
       load_and_authorize_resource
   
       ...
     end
   ```